bats/create-tables.bats

```diff
@@ -94,13 +94,13 @@ CREATE TABLE test1 (
 SQL
     dolt sql <<SQL
 CREATE TABLE test2 (
-  pk1 BIGINT NOT NULL COMMENT 'tag:0',
-  pk2 BIGINT NOT NULL COMMENT 'tag:1',
-  c1 BIGINT COMMENT 'tag:2',
-  c2 BIGINT COMMENT 'tag:3',
-  c3 BIGINT COMMENT 'tag:4',
-  c4 BIGINT COMMENT 'tag:5',
-  c5 BIGINT COMMENT 'tag:6',
+  pk1 BIGINT NOT NULL COMMENT 'tag:10',
+  pk2 BIGINT NOT NULL COMMENT 'tag:11',
+  c1 BIGINT COMMENT 'tag:12',
+  c2 BIGINT COMMENT 'tag:13',
+  c3 BIGINT COMMENT 'tag:14',
+  c4 BIGINT COMMENT 'tag:15',
+  c5 BIGINT COMMENT 'tag:16',
```

Comment:
Bats tests needed to be updated to either
1) Eliminate a tag conflict created by a tag annotation
2) Add a tag annotation to prevent random tag generation